tests/Functional/Action/Lti/UpdateLti1p1OutcomeActionTest.php

```diff
@@ -239,7 +241,7 @@ private function getLtiInstance(): LtiInstance
         /** @var LtiInstanceRepository $repository */
         $repository = $this->getRepository(LtiInstance::class);
 
-        $ltiInstance = $repository->find(1);
+        $ltiInstance = $repository->find(new UuidV6('00000001-0000-6000-0000-000000000000'));
```

Comment:
Why do we really need two different UUID types in here?

Reply:
What do you mean sorry?

Reply:
Actually it was related to same reason as Luis mentioned - about having two different UUID types used. But you've already answered it ;)